ios-involves/ios-involves/Application/AppDelegate.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AppDelegate.swift
+//  ios-involves
+//
+//  Created by Danilo Pena on 20/02/20.
+//  Copyright © 2020 Danilo Pena. All rights reserved.
+//
+
+import UIKit
+import TraktKit
+
+extension Notification.Name {
+    static let TraktSignedIn = Notification.Name(rawValue: "TraktSignedIn")
+}
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
```suggestion
final class AppDelegate: UIResponder, UIApplicationDelegate {
```
É importante sempre colocar `final` se possível